input/pagecontent/general-requirements.md

```diff
@@ -18,246 +18,74 @@ Implementers are advised to be familiar with the requirements of the FHIR standa
-
-
-**Declaring Conformance**
-
-Servers that implement Profile Only Support declare conformance to each profile supported:
-* The server hosts a [capability statement](http://hl7.org/fhir/capabilitystatement.html) at [url]/metadata that is available to both authenticated and unauthenticated clients that declares the profiles supported as either a base profile using [CapabilityStatement.rest.resource.profile](http://hl7.org/fhir/capabilitystatement-definitions.html#CapabilityStatement.rest.resource.profile) or supported profile using [CapabilityStatement.rest.resource.supportedProfile](http://hl7.org/fhir/capabilitystatement-definitions.html#CapabilityStatement.rest.resource.supportedProfile).
-
-    Example: CapabilityStatement resource for a server supporting the AU Core Patient profile as a system-wide profile that is applied across all instances of the Patient resource:
-
-    ```
-    {
-      "resourceType": "CapabilityStatement",
-      ...
-      "instantiates": [
-        "http://hl7.org.au/fhir/core/CapabilityStatement/au-core-server"
-      ],
-      ...  
-      "rest": [
-        {
-          "mode": "server",
-        ...
-            "resource": [
-              ...
-              {
-                "type": "Patient",
-                "profile": [
-                  "http://hl7.org/fhir/us/core/StructureDefinition/au-core-patient"
-                ],
-                ...
-              }
-            ]
-        }
-      ] 
-    }
-    ```
-
-    Example: CapabilityStatement resource for a server supporting the AU Core Waist Circumference profile as a profile for supported use cases provided by the server:
-
-    ```
-    {
-      "resourceType": "CapabilityStatement",
-      ...
-      "instantiates": [
-        "http://hl7.org.au/fhir/core/CapabilityStatement/au-core-server"
-      ],
-      ...  
-      "rest": [
-        {
-          "mode": "server",
-        ...
-            "resource": [
-              ...
-              {
-                "type": "Observation",
-                "supportedProfile": [
-                  "http://hl7.org.au/fhir/core/StructureDefinition/au-core-waistcircum"
-                ],
-                ...
-              }
-            ]
-        }
-      ] 
-    }
-    ```
+A system that [declares conformance](declaring-conformance.html#profile-only-support) as _Profile Only Support_ to an AU Core profile:
+- **SHALL** correctly populate a resource with all mandatory elements specified in the profile’s definition.
+- **SHALL** correctly populate a resource with all _Must Support_ elements if a value is known for the element. 
+- **SHALL** implement the requirements on [Missing data](general-requirements.html#missing-data) when an element value is not known.
```

Comment:
Change link text "Missing data" to "Missing Data" to make it consistent with the section title.

Reply:
fixed.